src/components/Firebase.jsx

```diff
@@ -63,21 +65,28 @@ export async function updateCommitmentFirestore(owner_address, account_id, commi
     const key2 = `ownedCDPs.${cdp_address}.commitmentTimestamp`
     try {
         const walletRef = doc(db, "users", owner_address);
-        const docRef = await updateDoc(walletRef, {
+        await updateDoc(walletRef, {
             [key1]: commitment_amt,
             [key2]: Date.now(),
+            webappActions: arrayUnion({actionType: 4, cdpAddress: cdp_address, microAlgos: 0, 
+            microGARD: 0, microGAIN: 0, swapPair: 0, feesPaid: 1000, timestamp: Date.now()})
         });
     } catch (e) {
         console.error("Error adding document: ", e);
     }
 }
-export async function updateLiquidationFirestore(owner_address, account_id) {
+export async function updateLiquidationFirestore(account_id) {
+    const owner_address = getWalletInfo().address
```

Comment:
The owner of the liquidated CDP is not the person who bid on it in our auction, so it should be passed as an argument instead of retrieved like this